CHANGES.md

```diff
@@ -3,6 +3,8 @@
 ## Unreleased
 
 * Add new entries here
+* Approximately 40% faster according to `cargo bench`.
+* Some "numeric" characters like `¾` and `①` were being treated as digits.
```

Comment:
I cannot create an example that would be effected at this time.  The tokenizer was returning None when encountering a character that was numeric but not a digit, however the parser conveniently avoided the problem by treating the None as an end of statement. It may be fine to remove it from the changelog.